modules/cnv-vm-wizard-fields-web.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/TBD
+[id="cnv-vm-wizard-fields-web_{context}"]
```

Comment:
This information should be presented in the UI, not in a table. If it must be in the docs, the content should be presented in the steps in the procedure.

Reply:
The table isn't rendering right. If you keep it, build it locally and confirm that the layout's correct.